CheckoutEventLoggerKit.xcframework/ios-arm64_armv7/CheckoutEventLoggerKit.framework/Modules/CheckoutEventLoggerKit.swiftmodule/arm.swiftinterface

```diff
@@ -1,16 +1,16 @@
 // swift-interface-format-version: 1.0
-// swift-compiler-version: Apple Swift version 5.4.2 (swiftlang-1205.0.28.2 clang-1205.0.19.57)
+// swift-compiler-version: Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)
```

Comment:
I did use xcode 12.3 may be I will have to use xcode 13.1 for this

Reply:
yes you do